x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -54,6 +54,7 @@ export interface AddCommentRefObject {
   editor: EuiMarkdownEditorRef | null;
 }
 
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
I think this is a false positive, right? It complains about `onCommentPosted` not being used but it is used.

Reply:
Yeah as far as I understand, it can't always detect if they are being used or not. So as far as I know, using this ignore-comment is the only reasonable way to get around it